pkg/kube/inject/monitoring.go

```diff
@@ -38,6 +38,12 @@ var (
 		"sidecar_injection_skip_total",
 		"Total number of skipped sidecar injection requests.",
 	)
+
+	injectionTime = monitoring.NewDistribution(
+		"sidecar_injection_time_seconds",
+		"Total time taken for injection in seconds.",
+		[]float64{.1, 1, 3, 5, 10, 20, 30, 60},
```

Comment:
60 seems too long

Reply:
What do you suggest?

Reply:
> Default timeout for a webhook call is 10 seconds, You can set the timeout and it is encouraged to use a short timeout for webhooks. If the webhook call times out, the request is handled according to the webhook's failure policy.

IMO, `10s` is enough, WDYT?

Reply:
We are actually running in to timeouts for some of the large manifests. So wanted to measure how much it is taking by increasing the timeout. That is the main intent of this change.

Reply:
make sense

Reply:
Does it really matter if its 10s or 60s? Either way its super broken

If you are seeing >1s injection time we should fix that instead of making the timeout 60s probably. There is no reason a json template+merge should take 60s IMO.

Fine with the metric though

Reply:
yes. We are internally getting the data/reproducible test case to see what is going on. Will fix the actual problem as well